tuftool/src/common.rs

```diff
@@ -0,0 +1,28 @@
+/// This module if for code that is re-used by different `tuftool` subcommands.
+use crate::error::{self, Result};
+use snafu::ResultExt;
+use std::fs::File;
+use std::path::Path;
+use tough::{Repository, Settings};
+
+/// Some commands only deal with metadata and never use a targets directory.
+/// When loading a repo that does not need a targets directory, we pass this as
+/// the targets URL.
+pub(crate) const UNUSED_URL: &str = "file:///unused/url";
+
+/// Load a repo for metadata processing only. Such a repo will never use the
+/// targets directory, so a dummy path is passed.
```

Comment:
Might also be worth noting that it's only for local file paths -- maybe even renaming the function? -- it seems too easy to pass a URL as a string, which is a valid `AsRef<Path>`.

Reply:
Addressed in [59899dd](https://github.com/awslabs/tough/compare/3e858e5b5d629576a0c13f3f7b627cd8c49af9a4..59899ddf735ceb7e4edc883a9b1c6a195506f820),